docs/gatekeeper-policy-authoring.md

```diff
@@ -0,0 +1,254 @@
+# Gatekeeper Policy Authoring
+Ratify can be deployed using passthrough execution mode behind admission
```

Comment:
suggestion to add all possible execution mode, "default", "passthrough" , emtpy etc

Reply:
agreed. I think in general it might be a bit confusing to a user to understand what the purpose of the built in policy provider is and how it works in comparison to the pass through mode

Reply:
Added #260 to cover adding to the documentation in the future.